ui/components/app/connected-accounts-permissions/connected-accounts-permissions.component.js

```diff
@@ -64,20 +64,25 @@ export default class ConnectedAccountsPermissions extends PureComponent {
         >
           <p>{t('authorizedPermissions')}:</p>
           <ul className="connected-accounts-permissions__list">
-            {permissions.map(({ key: permissionName }) => (
-              <li
-                key={permissionName}
-                className="connected-accounts-permissions__list-item"
-              >
-                <CheckBox
-                  checked
-                  disabled
-                  id={permissionName}
-                  className="connected-accounts-permissions__checkbox"
-                />
-                <label htmlFor={permissionName}>{t(permissionName)}</label>
-              </li>
-            ))}
+            {permissions.map(({ key }) => {
+              const permissionName =
+                key === 'eth_accounts' ? 'permission_ethereumAccounts' : key;
```

Comment:
I kept the arbitrary string(s) here for now. We might want to create reusable constant(s) here. 

We have constants for "RestrictedMethods" and I would need more time to understand the correlation between permissions and the related restricted methods.